.vale.ini

```diff
@@ -0,0 +1,22 @@
+; Path to the "vale" styles
+StylesPath = .github/styles
+
+; Extra "vale" packages to download
+Packages = Google, proselint
```

Comment:
Can't it simply download these? Why would we need to add the source to the repo directly?

Reply:
Yes, I checked the documentations right now & had missed it earlier. The packages needn't be added to VCS & even the official docs [recommends ignoring](https://vale.sh/docs/topics/packages/#packages-and-vcs) those. The contents of the `.github/styles/Vocab` directory has to be version-controlled though. That directory contains lists of specific vocabularies which Vale recognises as "correct" like the word "Starlite" & so on.

On that note, I believe, ignoring the styles from version control will allow the CodeSpell job in the CI to pass well, so yeah thanks for the heads up!